README.md

```diff
@@ -1 +1,205 @@
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails with React <a name="about-project"></a>
+
+**Rails with React** is a fullstack app that have 5 greetings
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+    <li><a href="https://reactjs.org/">React</a></li>
+</details>
+<details>
+  <summary>Server</summary>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+</details>
+<details>
+  <summary>Database</summary>
+    <li><a href="https://www.postgresql.org/">Postgres</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Built using Ruby on Rails with React**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Link To React App <a name="live-demo"></a>
+
+- [Link to React App PR](https://github.com/tajulafreen/hello-react-front-end/pull/1)
+- [Link to React App feat/UI](https://github.com/tajulafreen/hello-react-front-end/tree/setup)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby >= 3.2.0
+    rails >= 7.0
+    postgres >- 15.3
+    node
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/tajulafreen/hello-rails-back-end/
+```
+
+You need to setup database for these project
+
+```
+  development = your_dataabase_dev
+  test        = your_datasebase_test
+  production  = your_datasebase_prod
+```
+
+or you can use your own database and change the ```config/database.yml```
+
+```yml
+  default: &default
+    adapter: postgresql
+    encoding: unicode
+    pool: 5
+    username: [your_username]
+    password: [your_password]
+    host: localhost
+
+  development:
+    <<: *default
+    database: [your_database_for_development]
+
+  test:
+    <<: *default
+    database: [your_database_for_test]
+
+  production:
+    <<: *default
+    database: [your_database_for_production]
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd hello-rails-back-end
+  bundle install
+  npm install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+to use this project:
+
+```bash
+   npm run build
+   bin/rails server
+```
+
+it will run the the server on ```localhost:3000```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="author"></a>
+
+👤 **tajulafreen**
+
+- GitHub: [@tajulafreen](https://github.com/tajulafreen)
+- Linkedin: [@tajulafreen](https://www.linkedin.com/tajul-afreen)
```

Comment:
- [ ] Kindly fix the link to your LinkedIn page, when I click the link it says page not found 👍🏿